airflow/models/dagbag.py

```diff
@@ -352,9 +352,9 @@ def _process_modules(self, filepath, mods, file_last_changed_on_disk):
                     dag.fileloc = filepath
             try:
                 dag.is_subdag = False
-                self.bag_dag(dag=dag, root_dag=dag)
                 if isinstance(dag.normalized_schedule_interval, str):
                     croniter(dag.normalized_schedule_interval)
+                self.bag_dag(dag=dag, root_dag=dag)
```

Comment:
Can we add a simple test case for this, i.e. to show what happens when a garbage value is provided to schedule-interval

Reply:
@kaxil found that such a testcase already exists, so added a check to assert the dagbag size on top of it.